examples/dense_bubble.py

```diff
@@ -118,10 +122,10 @@
 
     bcs = [DirichletBC(Vu, 0.0, "bottom"),
            DirichletBC(Vu, 0.0, "top")]
-    diffused_fields = [("u", InteriorPenalty(state, Vu, kappa=Constant(75.),
-                                             mu=Constant(10./delta), bcs=bcs)),
-                       ("theta", InteriorPenalty(state, Vt, kappa=Constant(75.),
-                                                 mu=Constant(10./delta)))]
+    diffused_fields = [("u", InteriorPenalty(state, Vu, kappa=75.,
+                                             mu=10./delta, bcs=bcs)),
+                       ("theta", InteriorPenalty(state, Vt, kappa=75.,
+                                                 mu=10./delta))]
```

Comment:
Here's an example where you would use `Constant`, because `delta` changes through the loop.